app/controllers/application_controller.rb

```diff
@@ -6,6 +6,8 @@ class ApplicationController < ActionController::Base
   helper_method :current_user
   helper_method :assignments_count
 
+  protect_from_forgery with: :null_session
```

Comment:
I think this may make it harder for us to log if a CSRF attack attempt has happened as it stops the exception being raised - lets chat